nextjs-logs-manager/app/api/log/route.ts

```
@@ -0,0 +1,70 @@
+    const total = await prisma.log.count({
+        where: {
+            status,
+            service,
+            date: {
+                gte: initDate,
+                lte: endDate
+            }
+        }
+    })
+
+    if (isNaN(limit) || limit <= 0) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)